src/foo_listenbrainz2.h

```diff
@@ -47,6 +47,15 @@ namespace lbz
 		return regex_search(mbid, rx);
 	}
 
+	static bool is_valid_token(const char* mbid)
```

Comment:
I'd not name the parameter `mbid`, that confused me at first. I would expect a MusicBrainz UUID. Let's just call it e.g. `token`.

Reply:
Sure!